highfidelity/hifi
# WL#20856 Added Parenting Hotkeys to Edit.js Among Visual tweaks

https://worklist.net/20856

- Moved Audioscope hotkey from Modifiers + P to Modifiers + F2
- Added Window.notify to notifications.js for Generic non-error notifications (NotificationType.UNKNOWN)
- Added New Edit.js Advanced Menu options for Parenting / Unparenting Entities
- Added Hotkeys for Ctrl + P Parent and Ctrl + Shift + P to Unparent Entities. This should work even if the Edit Window is selected
- Added notification when User parents and unparents Entities via hotkeys
- Added Visual Indicators to Last Selected Entity - Last Entity will be bright yellow instead of orange
- Automatic Trailing Whitespace clean up on save of entitySelectionTool.js (sorry, this was accidental, I can revert this if needed)

To Test:

1. Rez a few entities, and open up edit mode.
2. Select the multiple entities. The Last entity you select should have a Yellow box instead of orange
3. Use  Ctrl - P to Parent objects
4. Observe Top Right popup
5. Check that entities are parented
6. Use Ctrl - Shift- P to Unparent entities
7. Observe Top Right popup
8. Check that entities are unparented
9. Repeat 3-8 while you select the edit window in an position
9. Repeat 3-8 with Advanced Menu only
